my_list.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
```

Comment:
- [ ] It will be great to ensure that you create your own enumerable `each` method. What you are doing now is using the method `each` which is already created for you in Ruby. Kindly create your own. Don't use the one created for you in the ruby language. Remember the point of this exercise is to implement your own emunarables. 

EXAMPLE: 
```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```